tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/constant/ConstantExpressionTest.java

```diff
@@ -59,6 +59,14 @@ public void constExpressionNegative() {
         BAssertUtil.validateError(compileResult1, i++, "operator '~' not defined for 'boolean'", 33, 22);
         BAssertUtil.validateError(compileResult1, i++, "operator '-' not defined for 'boolean'", 35, 21);
         BAssertUtil.validateError(compileResult1, i++, "operator '~' not defined for 'boolean'", 37, 22);
+        BAssertUtil.validateError(compileResult1, i++, "incompatible types: expected '3', found 'int'", 49, 15);
```

Comment:
Shall we move all of these newly added tests to a separate test class? Like `ConstantTypeTest`? The existing tests in the current tests seem to be testing something else, right?